README.md

```diff
@@ -220,25 +220,28 @@ following entry to your `.sync.yml` and run `pdk update`.
 
 ```yaml
 Gemfile:
-  ':system_tests':
-    - gem: 'puppet-module-posix-system-r#{minor_version}'
-      platforms: ruby
-    - gem: 'puppet-module-win-system-r#{minor_version}'
-      platforms:
-        - mswin
-        - mingw
-        - x64_mingw
-    - gem: beaker
-      version: '~> 3.13'
-      from_env: BEAKER_VERSION
-    - gem: beaker-abs
-      from_env: BEAKER_ABS_VERSION
-      version: '~> 0.1'
-    - gem: beaker-pe
-    - gem: beaker-hostgenerator
-      from_env: BEAKER_HOSTGENERATOR_VERSION
-    - gem: beaker-rspec
-      from_env: BEAKER_RSPEC_VERSION
+  required:
+    ':system_tests':
+      - gem: 'puppet-module-posix-system-r#{minor_version}'
+        platforms: ruby
+      - gem: 'puppet-module-win-system-r#{minor_version}'
+        platforms:
+          - mswin
+          - mingw
+          - x64_mingw
+      - gem: beaker
+        version: '~> 3.13'
+        from_env: BEAKER_VERSION
+      - gem: beaker-abs
+        from_env: BEAKER_ABS_VERSION
+        version: '~> 0.1'
+      - gem: beaker-pe
+      - gem: beaker-hostgenerator
+        from_env: BEAKER_HOSTGENERATOR_VERSION
+      - gem: beaker-rspec
+        from_env: BEAKER_RSPEC_VERSION
+.travis.yml:
+  bundler_args: --with system_tests
```

Comment:
We should probably add the equivalent settings for Appveyor and Gitlab-CI configs.

Reply:
@scotje like that?

Reply:
Yep! It looks like this has uncovered a bug in the gitlab customization though, I'm still working through that. Also it looks like the config for `.gitlab-ci.yml` has a `beaker` key that we might also want to suggest setting?

Reply:
@scotje added `beaker: true` I wonder where one could fine a more complete example showing the use of `bekaer.tags` and `beaker.variables`